src/main/java/it/eng/idsa/dataapp/util/MessageUtil.java

```diff
@@ -0,0 +1,49 @@
+package it.eng.idsa.dataapp.util;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.springframework.http.HttpHeaders;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class MessageUtil {
+	
+	public static String createResponsePayload() {
+		// Put check sum in the payload
+		DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
+		Date date = new Date();
+		String formattedDate = dateFormat.format(date);
+		
+		 Map<String, String> jsonObject = new HashMap<>();
+         jsonObject.put("firstName", "John");
+         jsonObject.put("lastName", "Doe");
+         jsonObject.put("dateOfBirth", formattedDate);
+         jsonObject.put("address", "591  Franklin Street, Pennsylvania");
+         jsonObject.put("checksum", "ABC123 " + formattedDate);
+         Gson gson = new GsonBuilder().create();
+         return gson.toJson(jsonObject);
+	}
+	
+	public static HttpHeaders createHttpHeaderResponseHeaders() {
```

Comment:
Can we move this one to HttpHeader controller? Do not see that it is used anywhere else but in that Controller.
Or at least to that MultipartMessageService, where we create HttpEntity for form and mix.

Reply:
Yes we can since we don't use it anywhere else at the moment. I putt it there in case we would need it somewhere else in the future.